Automattic/wp-calypso
# Editor welcome tour: Remove editor welcome tour images and replace with images urls from the CDN

The editor welcome tour images are now available from the CDN. This PR removes the slide images (gif and png files) from the repo, and updates the urls to point to the CDN.

#### Changes proposed in this Pull Request

* Remove gif and png editor welcome tour slide images from the repo
* Update urls to point to the CDN

#### Testing instructions



* Apply the diff associated with this PR
* Enable the welcome tour on a sandboxed site (e.g. set `define( 'SHOW_WELCOME_TOUR', true );` in your `0-sandbox.php`) file
* Open up the editor on your sandboxed site and display the welcome tour (more menu, select Welcome Guide under Tools)
* On each of the slides of the welcome tour, right click and inspect element on the image of the slide, and make sure the img src is pointing to a s0.wp.com url. For example:

![image](https://user-images.githubusercontent.com/14988353/104252805-1146a680-54c7-11eb-91b2-d35b2ace8bba.png)

* Also, please double-check that I haven't missed any stray binary images!

Part of https://github.com/Automattic/wp-calypso/issues/48751
